CatNote.Tests/Services/AchievementGenericServiceTests.cs

```diff
@@ -0,0 +1,221 @@
+﻿using AutoMapper;
+using CatNote.BLL.Models;
+using CatNote.BLL.Services;
+using CatNote.DAL.Entities;
+using CatNote.DAL.Interfaces;
+using CatNote.Tests.Services.DataForTests;
+using FluentAssertions;
+using Moq;
+using Xunit;
+
+namespace CatNote.Tests.Services;
+
+public class AchievementGenericServiceTests
+{
+    private readonly Mock<IGenericRepository<AchievementEntity>> _mockGenericRepository;
+    private readonly Mock<IMapper> _mockMapper;
+    private readonly GenericService<AchievementModel, AchievementEntity> _achievementService;
+
+    public AchievementGenericServiceTests()
+    {
+        _mockGenericRepository = new Mock<IGenericRepository<AchievementEntity>>();
+        _mockMapper = new Mock<IMapper>();
+        _achievementService = new GenericService<AchievementModel, AchievementEntity>(_mockMapper.Object, _mockGenericRepository.Object);
+    }
+
+    [Fact]
+    public async Task Create_CorrectModelPass_CallCreateMethodOfRepositoryAndReturnAchievementModel()
```

Comment:
Long test names. Simplify according to Given-When-Then.
For ex: Create_WhenModelIsCorrect_ReturnsAchievementModel